spec/system/category_topics_spec.rb

```diff
@@ -32,3 +32,24 @@
     expect(category_list).to have_no_new_posts_badge
   end
 end
+
+describe "Viewing category topic list", type: :system do
```

Comment:
I don't think it is common practice to have multiple top level describe blocks in a single system spec file. Can you see if you can combine it with the one above? Maybe something like:

```
describe "Viewing category page", type: :system do
  describe "when viewing top topics" do
    ...
  end

  describe "when viewing a parent category with `default_list_filter` set to `none`"
   end
end
```